subsys/net/openthread/Kconfig.defconfig

```diff
@@ -251,6 +251,15 @@ endif # OPENTHREAD_CSL_RECEIVER
 
 endif # !OPENTHREAD_THREAD_VERSION_1_1
 
+if OPENTHREAD_THREAD_VERSION_1_4
+
+# Thread 1.4 dependencies
+config OPENTHREAD_TCP_ENABLE
+	bool
```

Comment:
```suggestion
```

Reply:
Or ignore it since all others are done in the same way.

Reply:
Do not ignore, remove, do not re-declare existing types for symbols that already exist

Reply:
ok, removed.